solves PR 33

gilles437 opened the pull request:
As described here:
 https://github.com/paritytech/polkadot-runtime-prom-exporter/issues/33

gilles437 committed:
solves PR 33

As described here:
 https://github.com/paritytech/polkadot-runtime-prom-exporter/issues/33

gilles437 committed:
Merge branch 'master' into timescasledb-history

gilles437 committed:
updated tsdb schema

updated tsdb schema for new installation

gilles437 committed:
Merge branch 'timescasledb-history' of https://github.com/paritytech/polkadot-runtime-prom-exporter into timescasledb-history

kianenigma commented:
The diff seems to be too big. Perhaps you need to first push all the formatting changes, or this PR is based on another branch?

gilles437 committed:
solving PR35

solving PR35
https://github.com/paritytech/polkadot-runtime-prom-exporter/issues/35#issuecomment-1250738580

gilles437 commented:
Most of the diff are due to reformatting the code
